.github/templates/getting-sample-code.mdpp

```diff
@@ -1,7 +1,7 @@
 ## Getting the sample code
 
 Get the latest sample code from GitHub using Git or download the repository as a ZIP file.
-([Download](https://github.com/GoogleCloudPlatform/java-docs-samples/archive/master.zip))
+([Download](https://github.com/GoogleCloudPlatform/java-docs-samples/archive/main.zip))
```

Comment:
Wonder if this will be the right URL.

Reply:
Yeah I wasn't entirely sure about this one. I think it should work because we are renaming the branch not creating a new branch to replace the old one. That being said, we can leave in or out and update after we update the repo branch and test.